client/keys.go

```diff
@@ -13,11 +13,9 @@ import (
 
 // PrivateKey represents a keyless-backed RSA private key.
 type PrivateKey struct {
-	public crypto.PublicKey
-	ski    gokeyless.SKI
-	digest gokeyless.Digest
-
-	client *Client
+	public   crypto.PublicKey
+	client   *Client
+	template *gokeyless.Operation
```

Comment:
That's fair. It was mainly in order to be extensible with other/new `gokeyless.Operation` info.